bundler/spec/support/helpers.rb

```diff
@@ -432,6 +432,14 @@ def simulate_new_machine
       pristine_system_gems :bundler
     end
 
+    def simulate_ruby_platform(ruby_platform)
+      old = ENV["BUNDLER_SPEC_RUBY_PLATFORM"]
+      ENV["BUNDLER_SPEC_RUBY_PLATFORM"] = ruby_platform.to_s
+      yield if block_given?
+    ensure
+      ENV["BUNDLER_SPEC_RUBY_PLATFORM"] = old if block_given?
+    end
```

Comment:
This helper is never currently used with a block, right? I would remove the unused code paths if that's the case.

Actually, in my opinion the cleanest version of this helper is with a block, so I'd change the helper to only support passing a block and change the tests that use it to the block version.

Reply:
Yeah, this was copied from `simulate_platform`. Adjusted now.